### DISCUSSION TITLE
Struggling with the groups concept for recurring backups

### DISCUSSION THREAD

**Discussion by kenlasko** [longhorn/longhorn#5335] [Q&A] [answered]
Hi all, 
I've been migrating all my Docker containers to a 4-node K3S installation backed by Longhorn. Longhorn is working fantastically. I can backup and restore volumes as needed. I'm now trying to create a nightly backup schedule that only targets high-value volumes. I've read that I can use groups to accomplish this, but I'm not having any luck with determining how to assign a volume to a group. 

I presumably add a term such as "high-value" to the Groups section in the Recurring Backup edit screen. How do I set a volume to be part of the "high-value" group?

My apologies if I'm mis-understanding what this feature is for, or if I've missed some terribly obvious bit of info.

**Comment by kenlasko** [comment]
I figured it out. First you set the group name in your Recurring Backup. Then you go into the Schedule for each volume and set the Group there.  Then it will backup only those volumes. Now, how can I set this via YAML?

**Comment by weizhe0422** [comment, marked as answer]
IIUC, you have created a set of repetitive jobs and want to apply to some Longhorn volumes.

Then you can use the command below to achieve it.

`kubectl -n longhorn-system label volumes.longhorn.io <your-longhorn-volume-name> recurring-job-group.longhorn.io/<recurring-job-group-name>=enabled`

This is the official document for your reference: https://longhorn.io/docs/1.4.0/snapshots-and-backups/scheduling-backups-and-snapshots/#recurring-job-assignment-using-the-kubectl-label-command

**Reply by kenlasko** [reply]
This is perfect. I don't know why I didn't see this when looking through the docs myself. I hate asking questions that are answered by the very same docs I was looking at. 

Thank you very much.
